### PR TITLE
add more pulsar config for external pulsar

### PR DESCRIPTION
**Pull request by stone-wlg**
add more pulsar config for external pulsar

## Description
1. We want to use external pulsar or pulsar cluster to improve performance

### CODE REVIEW

File: helm-charts/FATE/templates/core/python-spark.yaml [hunk 1]
```diff
@@ -129,11 +129,6 @@ spec:
                 cp /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults-template.conf /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
                 sed -i "s/fateflow/${POD_IP}/g" /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
                 
-                # logs
-                mkdir -p /data/projects/fate/fateflow/logs/fate_flow/
-                touch /data/projects/fate/fateflow/logs/fate_flow/INFO.log
-                ln -sf /dev/stdout /data/projects/fate/fateflow/logs/fate_flow/INFO.log
-
```
Comment (JingChen23) [inline helm-charts/FATE/templates/core/python-spark.yaml:136]: https://github.com/FederatedAI/KubeFATE/commit/772e174dba04b161c493e8b8a844bdd9b371cb1c

It's you who removed these stuffs on May 10 and now you add them back, what is the story behind this?

Comment (stone-wlg) [reply helm-charts/FATE/templates/core/python-spark.yaml:136]: it means it is removed. i do not add them back.

Comment (JingChen23) [reply helm-charts/FATE/templates/core/python-spark.yaml:136]: Oops, my fault

**Comment by JingChen23** [comment]
Could you please pull the latest develop-1.10.0 branch, make your change and send pr? This one has conflicts which need to be handled manually.

**Pull request closed without merging**